src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -338,6 +337,7 @@ static std::vector<SenderThreadParams*>* buildSenderParamsCustom
     params->renderFormat     = renderFormatToString(renderFormat);
     params->fiwareCorrelator = fiwareCorrelator;
     params->extraHeaders     = headers;
+    params->registration     = false;
```

Comment:
> Probably soon registrations will have the same functionality as well and a variable with a very specific name will then be worthless

Well... actually I think that a most probable evolution is to remove comopletely the NGSI9 registration notification stuff. It is a mostly useless feature (nobody uses it). Thus, I think is improvable that NGSI9 registration notifications use cache some day.

In this same line, creating an enum for this would increase compleixty with little gain.

Let's go with the current name of the bool field. It is no so important, as long as we remember what it is and doesn't confuse ourselves if we need to change this part of the code in the future.

NTC